src/SFML/Audio/Miniaudio.cpp

```diff
@@ -29,5 +29,4 @@
 #pragma GCC diagnostic ignored "-Wnull-dereference" // Work around MinGW warning
 #endif
 // #define MA_DEBUG_OUTPUT // Uncomment to enable miniaudio debug output to console
-#define MINIAUDIO_IMPLEMENTATION
-#include <miniaudio.h>
+#include <miniaudio.c>
```

Comment:
I’d prefer we not change this file. I don’t think we need an additional translation unit to compile Miniaudio.

Reply:
This change will have to be made whether we like it or not. The author has already stated that `MINIAUDIO_IMPLEMENTATION` is deprecated and will no longer be supported in the future. Including `miniaudio.c` is the officially sanctioned method to do what we are already doing now in a future proof way.

Reply:
Is the expectation that miniaudio.c is going to change in the future to actually include the function definitions instead of merely being the two lines that it is now?